api/controllers/main.js

```diff
@@ -70,6 +70,12 @@ module.exports = {
     return res.render('content/contact.njk', context);
   },
 
+  features(req, res) {
```

Comment:
Could we grab the part from the examples function above that redirects for authed users? Looks like this part isn't available from the authenticated navbar.